Example/Views/Examples/VectiralCollectionScreen.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  CollectionScreen.swift
+//  Portal
+//
+//  Created by Argentino Ducret on 8/29/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+
+import Portal
+
+enum VerticalCollectionScreen {
+    
+    typealias Action = Portal.Action<Route, Message>
+    typealias View = Portal.View<Route, Message, Navigator>
+    
+    static func view(color: Color, searching: Bool, showFullName: Bool) -> View {
+        return View(
+            navigator: .main,
+            root: .stack(navigationBar()),
+            component: container(
+                children: [
+                    collection(
+                        properties: properties(
+                            itemsWidth: 150,
+                            itemsHeight: 150
+                        ) {
+                            let items: [CollectionItemProperties<Action>] = (0...1000).map { index in
+                                collectionItem(onTap: .none, identifier: "Item \(index)") {
+                                    return container(
+                                        children: [
+                                            label(text: showFullName ? "Item \(index)" : "\(index)")
+                                        ],
+                                        style: styleSheet { $0.backgroundColor = .red },
+                                        layout: layout {
+                                            $0.height = Dimension(value: 150)
+                                            $0.width = Dimension(value: 150)
+                                            $0.alignment = Alignment(items: .center)
+                                            $0.justifyContent = .center
+                                        }
+                                    )
+                                }
+                            }
+                            
+                            let refreshState: RefreshState<Action> = searching ? .searching : .idle(searchAction: .sendMessage(.search))
+                            
+                            let refreshProperties: RefreshProperties<Action> = properties(state: refreshState) {
+                                let title = NSAttributedString(string: "Collection refresh")
```

Comment:
Why do you need a local variable?

Reply:
if I do it in one line it doesn't compile. idk why.
I shared the error in slack.
For example If I use the next code, works perfectly:
``` swift code
let refreshProperties: RefreshProperties<Action> = properties(state: refreshState) {
      print()
      $0.title = NSAttributedString(string: "Collection refresh")
}
```

but if I do it in one line like this: 
``` swift code
let refreshProperties: RefreshProperties<Action> = properties(state: refreshState) {
      $0.title = NSAttributedString(string: "Collection refresh")
}
```
shows the next error, `Cannot assign to property: '$0' is immutable`

Reply:
Mm that sounds like a compiler error